docs/slides/innersource-summit.md

```diff
@@ -0,0 +1,237 @@
+# The InnerSource Program Office (ISPO) Working Group
+<!---  Slide 2: Opening Statement --->
+
+![bg right](../assets/innersource-commons.svg)
+
+The InnerSource Program Office (ISPO) Working Group develops guidance and products for enterprise InnerSource teams, not individuals or projects.
+
+<!--
+1. Define an ISPO
+2. ...
+-->
+--- 
+<!--- Slide 3 --->
+
+For those with InnerSource responsibilities in an OSPO, platform engineering, or other centralized group, this working group is for you.
+
+<!-- KEY MESSAGE 3:  -->
+
+--- 
+<!--- Slide 4 --->
+
+But what is an ISPO?
+
+"Person(s) working on improving the internal code collaboration experience at full organization scale"
+
+<!-- KEY MESSAGE 4:  
+
+Technically, ISPO is an acronym. InnerSource Programs Office. 
+Often however, the people doing ISPO related work in a company aren't in their own team. 
+They are part of an OSPO, part of a centralized engineering team, maybe they are single person working under a head engineering. 
+It is variable... and for our purpoeses though, it is just someone with InnerSource responsibilities (however you define that)
+be it on an OSPO, platform engineering, or any other centralized group.
+
+-->
+
+--- 
+
+<!--- Slide 5 --->
+
+## Lots of overlap
+OSPO <-> ISPO <-> Platform Engineering <-> Internal Engineering <-> Software Engineering Community of Practice
+
+You don't formally need an ISPO for this content to be useful
+
+<!-- KEY MESSAGE 5:  
+Introduce OSPO vs. ISPO. You don't formally need an ISPO for this content to be useful.
+
+-->
+
+--- 
+<!--- Slide 8 --->
+
+Q: WHY is the ISPO working group a subset of InnerSource Commons?
+
+A: Our focus on accelerating the teams that run and support InnerSource across an organization rather than guidance for projects or individuals
+
+<!-- KEY MESSAGE 8:  
+WHY: The scope of the InnerSource Programs Office (ISPO) working group is more focused on the teams that run and support InnerSource across an organization rather than guidance for projects or individuals
+-->
+
+
+
+--- 
+<!--- Slide 6 --->
+
+Our goal for this talk is to get people to learn more about the ISPO working group
```

Comment:
1. Removed for this talk since people know we're in a talk.
2. Made the object about helping the audience rather than about us getting people to learn.

```suggestion
Our objective is to help you discover the benefits of our ISPO working group.
```